python-modules/cffi-1.5.2/setup.py

```diff
@@ -1,6 +1,7 @@
 import sys, os
 import subprocess
 import errno
+import _ssl
```

Comment:
We don't need this particular one, but we'll need it if/when we'll get back to building CFFI on HP-UX. I judged it harmless, but I guess I should add comments for these extra imports.